ui/src/app/views/workspace-edit/component.ts

```diff
@@ -39,13 +39,17 @@ export class WorkspaceEditComponent implements OnInit {
         aggregateAnalysis: false,
         ancestry: false,
         commercialPurpose: false,
-        population: false
+        population: false,
+        reviewRequested: false
       }};
   }
 
   addWorkspace(): void {
```

Comment:
Is there a useful frontend test to add?

Reply:
I can add one which just includes adding a workspace with review requested. We don't actually currently have test suites for creating workspaces yet though, so that may be a good follow up PR.